src/Microsoft.AspNetCore.ResponseCompression/ResponseCompressionProvider.cs

```diff
@@ -38,7 +38,17 @@ public ResponseCompressionProvider(IServiceProvider services, IOptions<ResponseC
             if (_providers.Length == 0)
             {
                 // Use the factory so it can resolve IOptions<GzipCompressionProviderOptions> from DI.
-                _providers = new ICompressionProvider[] { new CompressionProviderFactory(typeof(GzipCompressionProvider)) };
+                _providers = new ICompressionProvider[]
+                {
+                    new CompressionProviderFactory(typeof(GzipCompressionProvider)),
+#if NETCOREAPP2_1
+                    new CompressionProviderFactory(typeof(BrotliCompressionProvider))
```

Comment:
We decided to make Brotli higher priority than Gzip. We'll have time to evaluate the impact in the preview releases. Please switch the order and then we can get this merged.